.github/workflows/RELEASE.yaml

```diff
@@ -1,11 +1,14 @@
-name: Build and Push Connector Runtime and Bundle Docker Image
+name: Build and Push Release Artifacts and Docker Images
 
 on:
   workflow_dispatch:
     inputs:
       version:
         description: 'Semantic version number of release: ^[0-9]+.[0-9]+.[0-9]+(-[a-zA-Z0-9.-]+){0,1}$'
         required: true
+      nextVersion:
+        description: 'Semantic version number to use as next SNAPSHOT version: ^[0-9]+.[0-9]+.[0-9]+(-[a-zA-Z0-9.-]+){0,1}$'
+        required: true
       release-runtime:
```

Comment:
Putting more thought into this, I really think it would be best to remove the Connector Runtime Docker image from this mono repository. It has no relation to the other things here, actually. For now, we'd have the Connector Runtime Docker repository and this monorepo for Connectors. At some point, we can merge the Connector Runtime Docker repository into Spring Zeebe (currently not possible to release the Docker image to Docker Hub from the Community Hub repository).

Reply:
https://github.com/camunda/connectors-bundle/issues/34

Reply:
Connector Runtime repository is ready to go. Runtime can be removed here.